go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/giantswarm/certs/v3 v3.1.1
 	github.com/giantswarm/ipam v0.2.0
 	github.com/giantswarm/k8sclient/v5 v5.11.0
-	github.com/giantswarm/k8scloudconfig/v10 v10.0.0
+	github.com/giantswarm/k8scloudconfig/v10 v10.0.1-0.20210223142608-908fd4f33c6a
```

Comment:
this one needs to be changed with the updated version of k8scloudconfig once https://github.com/giantswarm/k8scloudconfig/pull/879 is merged and a new release of k8scloudconfig has been pushed